sponsorship_compassion/models/contracts_report.py

```diff
@@ -150,6 +152,19 @@ def get_nb_meal(sponsorship):
                     get_nb_meal))
             partner.sr_nb_meal = total_meal
 
+    @api.multi
+    def _compute_medic_check(self):
+        def get_nb_check(sponsorship):
+            nb_year = sponsorship.contract_duration / 365
+            country = sponsorship.child_id.field_office_id
+            return nb_year * country.icp_medical_check
+
+        for partner in self:
+            total_check = sum(
```

Comment:
You forgot to include the church members in the count.

Reply:
No, they are include in related_sponsorships